src/pages/cluster/ClusterMemberMemoryUsage.tsx

```diff
@@ -0,0 +1,67 @@
+import type { FC } from "react";
+import { useQuery } from "@tanstack/react-query";
+import Meter from "components/Meter";
+import { fetchClusterMemberState } from "api/cluster-members";
+import { queryKeys } from "util/queryKeys";
+import { humanFileSize } from "util/helpers";
+import type { LxdClusterMember } from "types/cluster";
+
+interface Props {
+  member: LxdClusterMember;
+  className?: string;
+}
+
+const ClusterMemberMemoryUsage: FC<Props> = ({ member, className }) => {
+  const { data: state } = useQuery({
+    queryKey: [
+      queryKeys.cluster,
+      queryKeys.members,
+      member.server_name,
+      queryKeys.state,
+    ],
+    queryFn: async () => fetchClusterMemberState(member.server_name),
+  });
+
+  if (!state?.sysinfo) {
+    return <span className="u-text--muted">-</span>;
+  }
+
+  const totalMemory = state?.sysinfo?.total_ram ?? 0;
+  const freeMemory = state?.sysinfo?.free_ram ?? 0;
+  const sharedMemory = state?.sysinfo?.shared_ram ?? 0;
+  const bufferedMemory = state?.sysinfo?.buffered_ram ?? 0;
+
+  const usedMemory = totalMemory - freeMemory;
+
+  const memoryPercentage =
+    totalMemory > 0 ? (usedMemory / totalMemory) * 100 : 0;
+
+  const secondaryMemory = sharedMemory + bufferedMemory;
+  const secondaryPercentage =
+    totalMemory > 0 ? (secondaryMemory / totalMemory) * 100 : 0;
+
+  if (totalMemory === 0) {
+    return <span className="u-text--muted">No data</span>;
+  }
```

Comment:
Move this a bit up and we can remove the conditions above checking `totalMemory > 0 ? `